### ISSUE TITLE
mate-maximus crashes with BadWindow error

### ISSUE THREAD

**Issue by JimmyAx** [mate-desktop/mate-netbook#5]
When closing KeePassX when it's not having focus mate-maximus will crash. It might crash with other programs as well, but this is easiest to reproduce.

```
$ gdb mate-maximus 
GNU gdb (GDB) 7.4.1-debian
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>...
Reading symbols from /usr/bin/mate-maximus...Reading symbols from /usr/lib/debug/usr/bin/mate-maximus...done.
done.
(gdb) run
Starting program: /usr/bin/mate-maximus 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffef8fa700 (LWP 14553)]
no maximize: true
[New Thread 0x7fffef0f9700 (LWP 14554)]
The program 'mate-maximus' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadWindow (invalid Window parameter)'.
  (Details: serial 1732 error_code 3 request_code 20 minor_code 0)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the --sync command line
   option to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
[Thread 0x7fffef0f9700 (LWP 14554) exited]
[Thread 0x7fffef8fa700 (LWP 14553) exited]
[Inferior 1 (process 14550) exited with code 01]
```

**Comment by monsta** [comment]
I have no idea how this bug survived for so long, but anyway, it got a second life when we migrated to GTK+3. Reopening, this isn't fixed, and I can reproduce it with GTK+3.

For reference, Ubuntu bug report: https://bugs.launchpad.net/bugs/1615445.

**Comment by monsta** [comment]
Ok, this is now fixed, and the fix is in 1.18.1 and 1.16.2 releases.
